### PR TITLE
[351] Transform ChangeKind in ChangeDescription with the source of the change

### PR DESCRIPTION
**Pull request by sbegaudeau**

### CODE REVIEW

File: backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.collaborative.api.services;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * The change description is used to express the changes that have been performed in response to an input in the
+ * collaborative layer. It helps the various representations event processors among others determined if they are
```
Comment (pcdavid) [inline backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java:20]: Typo: "determine"

File: backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * impacted by a change and thus if they should refresh the representation that they are managing.
+ *
+ * @author sbegaudeau
+ */
+public class ChangeDescription {
```
Comment (pcdavid) [inline backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java:25]: A simple `toString` would be useful for debugging I think.

**Review by pcdavid — APPROVED** [review]

File: backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java (hunk 2 above)
Comment (sbegaudeau) [reply backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java:25]: Indeed, I'll add that

File: backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java (hunk 1 above)
Comment (sbegaudeau) [reply backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java:20]: Thanks

**Pull request merged**